src/components/Logo/Logo.jsx

```diff
@@ -0,0 +1,34 @@
+import React, {useEffect, useState} from 'react';
```

Comment:
How did @briankimlim  changes get into your branch?

Reply:
i pulled it to resolve conflicts early